runtime/queries/query/highlights.scm

```diff
@@ -63,6 +63,9 @@
   (comment) @keyword.directive)
   (#lua-match? @keyword.directive "^;+ *extends *$"))
 
+((comment) @keyword.directive
+  (#eq? @keyword.directive "; format-ignore"))
```

Comment:
Maybe more lenient?
```suggestion
  (#lua-match? @keyword.directive "^;%s*format-ignore"))
```

Reply:
But maybe more expensive? Now that we can rely on the format ;))

Reply:
The formatter right now is only checking if you have `format-ignore` in that comment or not :thinking: 
I think I will fixup the pattern inside the formatter to match this. It should be lenient to avoid unexpected results.